docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md

```diff
@@ -18,153 +18,255 @@ weight: 117
+```
+81.2345
+```
 
-If you were to use the `$value` variable in the summary of an alert:
+To print the value of a range query you must first reduce it from a time series to an instant vector with a reduce expression. You can then print the result of the reduce expression by using its Ref ID instead. For example, if the reduce expression takes the average of A and has the Ref ID B you would write:
 
 ```
-{{ $labels.service }} has over 5% of responses with 5xx errors: {{ $value }})
+{{ index $values "B" }}
 ```
 
-The summary might look something like the following:
+### Print the humanized value of a query
+
+To print the humanized value of an instant query use the `humanize` function:
 
 ```
-api has an over 5% of responses with 5xx errors: [ var='B' labels={service=api} value=6.789 ]
+{{ humanize (index $values "A").Value }}
 ```
 
-Here `var='B'` refers to the expression with the RefID B. In Grafana, all queries and expressions are identified by a RefID that identifies each query and expression in an alert rule. Similarly `labels={service=api}` refers to the labels, and `value=6.789` refers to the value.
+For example, given an instant query that returns the value 81.2345, this will print:
 
-You might have observed that there is no RefID A. That is because in most alert rules the RefID A refers to a query, and since queries can return many rows or time series they are not included in `$value`.
+```
+81.234
+```
 
-### The values variable
+To print the humanized value of a range query you must first reduce it from a time series to an instant vector with a reduce expression. You can then print the result of the reduce expression by using its Ref ID instead. For example, if the reduce expression takes the average of A and has the Ref ID B you would write:
 
-If the `$value` variable contains more information than you need, you can instead print the labels and value of individual expressions using `$values`. Unlike `$value`, the `$values` variable is a table of objects containing the labels and floating point values of each expression, indexed by their RefID.
+```
+{{ humanize (index $values "B").Value }}
+```
+
+### Print the value of a query as a percentage
 
-If you were to print the value of the expression with RefID `B` in the summary of the alert:
+To print the value of an instant query as a percentage use the `humanizePercentage` function:
 
 ```
-{{ $labels.service }} has over 5% of responses with 5xx errors: {{ $values.B }}%
+{{ humanizePercentage (index $values "A").Value }}
 ```
 
-The summary will contain just the value:
+This function expects the value to be a decimal number between 0 and 1. If the value is instead a decimal number between 0 and 100 you can either divide it by 100 in your query or using a math expression. If the query is a range query you must first reduce it from a time series to an instant vector with a reduce expression.
```

Comment:
```suggestion
This function expects the value to be a decimal number between 0 and 1. If the value is instead a decimal number between 0 and 100 you can divide it by 100 in your query or using a math expression. If the query is a range query you must first reduce it from a time series to an instant vector with a reduce expression.
```

Reply:
I think either should be kept here no?

Reply:
I think either should be kept here no? Without it the sentence doesn't quite read right.